src-tauri/src/lib.rs

```diff
@@ -294,9 +297,28 @@ pub fn launch_northstar(
     }
 
     // Only Windows with Steam or Origin are supported at the moment
-    if host_os == "windows"
-        && (matches!(game_install.install_type, InstallType::STEAM)
-            || matches!(game_install.install_type, InstallType::ORIGIN)
+    if matches!(game_install.install_type, InstallType::STEAM)
+    {
+        // Create run_northstar.txt for NorthstarProton
+        if host_os != "windows"
```

Comment:
Should `run_northstar.txt` also be used on Windows?
The codepath exists is only required on Linux to work around some issues (afaik)-

Reply:
Given that `-northstar` doesn't work a for a reason that is still unknown, it might be simpler to just use `run_northstar.txt` on both Windows and Linux. Otherwise you have to use `-northstar` on one and `run_northstar.txt` on the other ^^